src/ingest_validation_tools/table-schemas/includes/fields/transposition_transposase_source.yaml

```diff
@@ -0,0 +1,14 @@
+# bulkatacseq-v0.yaml; bulkatacseq-v1.yaml
+- name: transposition_transposase_source
+  description: The source of the Tn5 transposase and transposon used for capturing
+    accessible chromatin.
+
+
+# scatacseq-v0.yaml; scatacseq-v1.yaml
+- name: transposition_transposase_source
+  description: The source of the Tn5 transposase and transposon used for capturing accessible chromatin.
```

Comment:
description: The source of the Tn5 transposase used for capturing accessible chromatin.